Source/JavaScriptCore/CMakeLists.txt

```diff
@@ -151,31 +151,6 @@ set(JavaScriptCore_OBJECT_LUT_SOURCES
     wasm/js/WebAssemblyTagPrototype.cpp
 )
 
-list(APPEND JavaScriptCore_SOURCES
-    runtime/CachedTypes.cpp
-    ${JavaScriptCore_DERIVED_SOURCES_DIR}/JSCBytecodeCacheVersion.cpp
-    ${JavaScriptCore_DERIVED_SOURCES_DIR}/JSCBuiltins.cpp
-)
```

Comment:
This is removing `CachedTypes.cpp` and `JSCBuiltins.cpp` too. They are required.